{"uid":"abf0febb5e55d6a9","name":"test_the_tests","fullName":"tests.test_demo.Ayuda#test_the_tests","historyId":"d89e5d3dcae2f873575c0ac61689d19c","time":{"start":1657904731206,"stop":1657904734492,"duration":3286},"status":"failed","statusMessage":"AssertionError: assert 2 == 3","statusTrace":"self = <test_demo.Ayuda testMethod=test_the_tests>\n\n    def test_the_tests(self):\n        print(\"Corriendo test de prueba...\")\n        assert True\n    \n>       assert 2 == 3\nE       assert 2 == 3\n\ntests/test_demo.py:70: AssertionError","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_Ayuda","time":{"start":1657904722257,"stop":1657904722257,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: assert 2 == 3","statusTrace":"self = <test_demo.Ayuda testMethod=test_the_tests>\n\n    def test_the_tests(self):\n        print(\"Corriendo test de prueba...\")\n        assert True\n    \n>       assert 2 == 3\nE       assert 2 == 3\n\ntests/test_demo.py:70: AssertionError","steps":[],"attachments":[{"uid":"bbb1c15a9453e4b8","name":"stdout","source":"bbb1c15a9453e4b8.txt","type":"text/plain","size":36}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_Ayuda::0","time":{"start":1657904734538,"stop":1657904734538,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_demo"},{"name":"subSuite","value":"Ayuda"},{"name":"host","value":"fv-az453-292"},{"name":"thread","value":"2832-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_demo"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":6,"skipped":0,"passed":0,"unknown":0,"total":7},"items":[{"uid":"3a2dc4ea95bdb25d","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/30//#testresult/3a2dc4ea95bdb25d","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x5625dc63cb13 <unknown>\n#1 0x5625dc443688 <unknown>\n#2 0x5625dc467f1f <unknown>\n#3 0x5625dc4635aa <unknown>\n#4 0x5625dc49e64a <unknown>\n#5 0x5625dc4987a3 <unknown>\n#6 0x5625dc46e0ea <unknown>\n#7 0x5625dc46f225 <unknown>\n#8 0x5625dc6842dd <unknown>\n#9 0x5625dc6882c7 <unknown>\n#10 0x5625dc66e22e <unknown>\n#11 0x5625dc6890a8 <unknown>\n#12 0x5625dc662bc0 <unknown>\n#13 0x5625dc6a56c8 <unknown>\n#14 0x5625dc6a5848 <unknown>\n#15 0x5625dc6bfc0d <unknown>\n#16 0x7fec7524e609 <unknown>","time":{"start":1657904310521,"stop":1657904312095,"duration":1574}},{"uid":"ad1ffcd428820079","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/29//#testresult/ad1ffcd428820079","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55ccdb1a4b13 <unknown>\n#1 0x55ccdafab688 <unknown>\n#2 0x55ccdafcff1f <unknown>\n#3 0x55ccdafcb5aa <unknown>\n#4 0x55ccdb00664a <unknown>\n#5 0x55ccdb0007a3 <unknown>\n#6 0x55ccdafd60ea <unknown>\n#7 0x55ccdafd7225 <unknown>\n#8 0x55ccdb1ec2dd <unknown>\n#9 0x55ccdb1f02c7 <unknown>\n#10 0x55ccdb1d622e <unknown>\n#11 0x55ccdb1f10a8 <unknown>\n#12 0x55ccdb1cabc0 <unknown>\n#13 0x55ccdb20d6c8 <unknown>\n#14 0x55ccdb20d848 <unknown>\n#15 0x55ccdb227c0d <unknown>\n#16 0x7fe86e688609 <unknown>","time":{"start":1657903186502,"stop":1657903188024,"duration":1522}},{"uid":"179229099d4919e9","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/28//#testresult/179229099d4919e9","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x56439b02eb13 <unknown>\n#1 0x56439ae35688 <unknown>\n#2 0x56439ae59f1f <unknown>\n#3 0x56439ae555aa <unknown>\n#4 0x56439ae9064a <unknown>\n#5 0x56439ae8a7a3 <unknown>\n#6 0x56439ae600ea <unknown>\n#7 0x56439ae61225 <unknown>\n#8 0x56439b0762dd <unknown>\n#9 0x56439b07a2c7 <unknown>\n#10 0x56439b06022e <unknown>\n#11 0x56439b07b0a8 <unknown>\n#12 0x56439b054bc0 <unknown>\n#13 0x56439b0976c8 <unknown>\n#14 0x56439b097848 <unknown>\n#15 0x56439b0b1c0d <unknown>\n#16 0x7f7f3da3b609 <unknown>","time":{"start":1657896733644,"stop":1657896734633,"duration":989}},{"uid":"26a72630e0f8a979","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/27//#testresult/26a72630e0f8a979","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657839031743,"stop":1657839031744,"duration":1}},{"uid":"de7b908efc2383fb","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/26//#testresult/de7b908efc2383fb","status":"broken","statusDetails":"NameError: name 'driver' is not defined","time":{"start":1657838860891,"stop":1657838860891,"duration":0}},{"uid":"ae56afc0d717c14","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/25//#testresult/ae56afc0d717c14","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657838605942,"stop":1657838605943,"duration":1}}]},"tags":[]},"source":"abf0febb5e55d6a9.json","parameterValues":[]}